packages/cloud_security_posture/kibana/csp_rule_template/6664c1b8-05f2-5872-a516-4b2c3c36d2d7.json

```
@@ -23,10 +23,13 @@
                 "CIS 1.1.1",
                 "Control Plane Node Configuration Files"
             ],
-            "benchmark": {
-                "name": "CIS Kubernetes V1.23",
-                "version": "v1.0.0"
-            },
+            "benchmarks": [
+                {
+                    "name": "CIS Kubernetes V1.23",
+                    "id": "cis_k8s",
+                    "version": "v1.0.0"
+                }
+            ],
```

```suggestion
            "benchmarks": [
                {
                    "name": "CIS Kubernetes",
                    "benchmark_version: 1.23",
                    "id": "cis_k8s",
                    "rule_version": "1.0.0"
                }
            ],
```

i was trying to figure out why `version` is different than the one in the name, and figured we do this because the rego rule might change even if the benchmark didn't. maybe we should reflect that in the names if we're already changing a schema

---

as we've discussed, this fits CIS benchmark versioning
you can explore it [here](https://www.cisecurity.org/cis-benchmarks/#kubernetes)